README.md

```diff
@@ -36,43 +36,39 @@ The frame ID entry for the sent messages.
 
 `baudrate` (`int`, `921600`)
 
-The baud rate of the serial port. The available baud rates can be checked on the user manual. It is suggested that the baud rate is kept at `921600` to ensure the high frequency transimission. The device will send `permission denied` error code if the baud rate cannot support the desired data package at the desired frequency.
+The baud rate of the serial port. The available baud rates can be checked on the user manual. It is suggested that the baud rate is kept at `921600` to ensure the high frequency transmission. The device will send `permission denied` error code if the baud rate cannot support the desired data package at the desired frequency.
 
 `imu_rate` (`int`, `200`)
 
 The rate of the IMU data.
 
 ```
-enable_magnetic_field (bool, default: true)
-enable_pressure       (bool, default: true)
-enable_temperature    (bool, default: true)
+enable_magn  (bool, default: true)
+enable_press (bool, default: true)
+enable_temp  (bool, default: true)
 ```
 
 Enable other possible messages that the driver is available to send. Note that the frequency of the data for each of these messages will be the same with the IMU data, if the topic is enabled.
 
-`enable_sync_out` (`bool`, `true`)
+`sync_out_rate` (`int`, `20`)
```

Comment:
Is it `sync_out_rate` or `sync_rate`? And related to this, are you using an external clock to trigger cameras and time stamp the IMU or are you using the sync out of the imu to trigger the cameras?